frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeListPage/CodeLists/CodeLists.test.tsx

```diff
@@ -246,6 +246,8 @@ describe('CodeLists', () => {
 
   it('calls onDeleteCodeList when clicking delete button', async () => {
```

Comment:
Foreslår at vi også oppdaterer testbeskrivelsen slik at den stemmer overens med det vi tester.
```suggestion
  it('calls onDeleteCodeList when the user clicks the delete button and confirms', async () => {
```